contentcuration/contentcuration/models.py

```diff
@@ -2566,58 +2564,92 @@ def serialize_to_change_dict(self):
         return self.serialize(self)
 
 
-class TaskResultCustom(object):
-    """
-    Custom fields to add to django_celery_results's TaskResult model
-
-    If adding fields to this class, run `makemigrations` then move the generated migration from the
-    `django_celery_results` app to the `contentcuration` app and override the constructor to change
-    the app_label. See `0141_add_task_signature` for an example
-    """
+# class TaskResultCustom(object):
```

Comment:
I think we need a migration that drops the fields from the `TaskResult` model that were defined here?

Reply:
You might have to run something `python manage.py makemigrations django_celery_results` where the `django_celery_results` is the app name you're generating migrations for.